Backend/src/main/java/idealab/api/operations/PrintJobOperations.java

```diff
@@ -61,23 +75,25 @@ public PrintJobResponse newPrintJob(PrintJobNewRequest printJobNewRequest) {
         // TODO: Hash email so it is not in plaintext!!
         String emailHash = printJobNewRequest.getEmail();
         EmailHash databaseEmail = emailHashRepo.findByEmailHash(emailHash);
+        
         if (databaseEmail == null) {
             databaseEmail = new EmailHash(emailHash);
             databaseEmail = emailHashRepo.save(databaseEmail);
         }
 
         // Create customer record with email hash if it does not already exist
```

Comment:
I know, it's not your modification but could you also delete this comment ? Because it's so clear what it's doing. You should not comment  code scopes which are clear, easy to understand what they are doing. It's redundant. 

You should use comments when:

- Something it's not finished (i.e todos)
- Something is not clear (But you should try to write clean which is easy to understand you should probably do refactor instead of commenting. Since we are all lazy people we just go ahead with comments instead of refactoring :) )
- (It's still controversial) Some conditions required to be satisfied

“_Don’t comment bad code—rewrite it._” ~ Brian W. Kernighan and P. J. Plaugher

Some resources you might find helpful:

- [https://blog.usejournal.com/stop-writing-code-comments-28fef5272752](url)
- [http://apdevblog.com/comments-in-code/](url)
- [https://gist.github.com/wojteklu/73c6914cc446146b8b533c0988cf8d29](url)
- [https://www.todaysoftmag.com/article/1120/clean-code-comments-and-formatting](url)
- [https://hackernoon.com/how-to-write-clean-code-d557d998bb08](url)
- Our bible: [https://www.amazon.com/Clean-Code-Handbook-Software-Craftsmanship/dp/0132350882]

Also you should google  the clean code examples :)

NOTE: Also don't forget if you used too many comments in your code, it means you have written dirty code which is hard to understand. So you should probably refactor your code :)